apps/build.info

```diff
@@ -31,6 +31,10 @@ IF[{- !$disabled{apps} -}]
     SOURCE[openssl]=openssl.rc
   ENDIF
 
+  IF[{- !$disabled{deprecated} || $config{api} < 30000 -}]
```

Comment:
If you rebase, you should be able to change this to

```
IF[{- !$disabled{'deprecated-3.0'} -}]
```

Reply:
Fixed.